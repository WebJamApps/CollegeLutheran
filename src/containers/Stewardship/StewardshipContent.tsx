import ELCALogo from 'src/components/elcaLogo';
import parser from 'html-react-parser';
import { useContext } from 'react';
import { ContentContext } from 'src/providers/Content.provider';

const StewardshipContent = () => {
  const { content: { stewardshipPage } } = useContext(ContentContext);
  return (
    <div className="page-content">
      <div className="container-fluid">
        <p style={{ fontSize: '4pt', margin: '0' }}>&nbsp;</p>
        <div
          className="material-content elevation3"
          style={{
            maxWidth: '998px', margin: 'auto', minHeight: '74vh',
          }}
        >
          <h3 style={{ paddingTop: '22px', paddingBottom: '15px' }}>Stewardship</h3>
          <section style={{ marginTop: '20px', textAlign: 'left', marginBottom: '35px' }}>
            {parser(stewardshipPage && stewardshipPage.comments ? stewardshipPage.comments : '')}
          </section>
        </div>
        <ELCALogo />
      </div>
    </div>
  );
};
export default StewardshipContent;
