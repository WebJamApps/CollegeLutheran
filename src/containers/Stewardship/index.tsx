import DefaultContent from './StewardshipContent';
import commonUtils from '../../lib/commonUtils';

export function Stewardship(): JSX.Element {
  commonUtils.setTitleAndScroll('Stewardship', window.screen.width);
  return (<DefaultContent />);
}
