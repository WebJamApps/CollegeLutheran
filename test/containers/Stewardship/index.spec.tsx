/* eslint-disable @typescript-eslint/no-explicit-any */
import renderer from 'react-test-renderer';
import { Stewardship } from 'src/containers/Stewardship/index';

describe('Stewardship', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Stewardship />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
