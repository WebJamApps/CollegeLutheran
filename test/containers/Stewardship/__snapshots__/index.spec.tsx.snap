// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stewardship renders correctly 1`] = `
<div
  className="page-content"
>
  <div
    className="container-fluid"
  >
    <p
      style={
        {
          "fontSize": "4pt",
          "margin": "0",
        }
      }
    >
      Â 
    </p>
    <div
      className="material-content elevation3"
      style={
        {
          "margin": "auto",
          "maxWidth": "998px",
          "minHeight": "74vh",
        }
      }
    >
      <h3
        style={
          {
            "paddingBottom": "15px",
            "paddingTop": "22px",
          }
        }
      >
        Stewardship
      </h3>
      <section
        style={
          {
            "marginBottom": "35px",
            "marginTop": "20px",
            "textAlign": "left",
          }
        }
      />
    </div>
    <div
      style={
        {
          "marginLeft": "-9px",
          "textAlign": "center",
        }
      }
    >
      <a
        href="http://www.elca.org/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="ELCA LOGO"
          id="elcaLogo"
          src="https://dl.dropboxusercontent.com/s/wkzubcmmm3pqst4/elca-logo.png?dl=0"
          style={
            {
              "margin": "auto",
              "marginLeft": "",
              "paddingTop": "30px",
              "width": "308px",
            }
          }
        />
      </a>
    </div>
  </div>
</div>
`;
